README.md

```diff
@@ -86,25 +96,28 @@ $ cargo run --example name_of_example
 
 All available examples are listed under [examples][exr].
 
-Our most advanced example is currently called pong. It is a pong game, as you may have guessed it.
+Our most advanced example is currently called pong. It is a pong game, as you
+may have guessed it.
```

Comment:
We probably should have removed this completely a while ago. lol

Reply:
True. Perhaps it would be better to link to the [Games made with Amethyst](https://community.amethyst-engine.org/t/games-made-with-amethyst/134/2) forum post? Or just remove this line entirely?

Reply:
I think linking would be the best :)

Reply:
Sounds good to me!